about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>About Game Developer's Conference</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="assets/css/style.css">
+  </head>
+  <body>
+    <header class = "shared_header">
+      <!--Dynamic header content loaded from JS DOM-->
+    </header>
+    <main class="parent_main">
+        <section class="about_header">
+          <div class="welcome-text col-12 pt-5 mt-5">
+            <h3 class="text-left text-md-center hello_intro orange">"Hello Devs!"</h3>
+            <h1 class="text-uppercase text-left text-md-center font-weight-bold text-img-effect">GAME DEVELOPERS GLOBAL SUMMIT 2020</h1>
```

Comment:
- [ ] **Required:** On the mobile view, kindly center theses headers.

Your Design | Template 
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/152193810-310c5cbf-f8be-49a4-aab3-988616eaf475.png) | ![image](https://user-images.githubusercontent.com/27709832/152193987-b5f1226e-6107-4ff7-bd2a-cc23a2fa243c.png)